.github/workflows/build.yaml

```diff
@@ -11,8 +11,11 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest] 
-        arch: [x86_64, arm64] 
+        include:
+        - {os: ubuntu-latest, arch: aarch64}
+        - {os: ubuntu-latest, arch: x86_64}
```

Comment:
you'll need to pass this along as CIBW_ARCH and on linux-aarch64 you'll need to set up qemu as well